src/core/Akka.MultiNodeTestRunner.Shared/Sinks/Spec.cs

```diff
@@ -59,15 +59,12 @@ public SpecFail(int nodeIndex, string nodeRole, string testDisplayName) : base(n
             FailureMessages = new List<string>();
             FailureStackTraces = new List<string>();
             FailureExceptionTypes = new List<string>();
-            Timestamp = DateTime.UtcNow;
```

Comment:
This was hiding the `Timestamp` property I later added to `SpecPass`, which is the base class of `SpecFail` - I added them the other way around the first time, so I wanted to clean that up here.